readme.md

```diff
@@ -216,6 +216,8 @@ Type: [`stream.Readable`](https://nodejs.org/api/stream.html#stream_class_stream
 - [`ics`](https://en.wikipedia.org/wiki/ICalendar#Data_format) - iCalendar
 - [`glb`](https://github.com/KhronosGroup/glTF) - GL Transmission Format
 - [`pcap`](https://wiki.wireshark.org/Development/LibpcapFileFormat) - Libpcap File Format
+- [`voc`](https://wiki.multimedia.cx/index.php/Creative_Voice) - Creative Voice File
+- [`ac3`](https://www.atsc.org/standard/a522012-digital-audio-compression-ac-3-e-ac-3-standard-12172012/) - ATSC A/52 Audio File
```

Comment:
There is indeed no good wiki page for that, https://en.wikipedia.org/wiki/Dolby_AC-3 is redirected to Dolby Digital which is in my opinion simply incorrect.

Reply:
Well, it's official standard. Best place to link I guess, unless link stops working.

Reply:
Fine with me, I just wanted to say, checked.